polybar-scripts/inbox-reddit/inbox-reddit.sh

```diff
@@ -1,10 +1,31 @@
 #!/bin/sh
+#
+# This script retrieves unread messages from your Reddit account using a JSON
+# link.
+#
+# To use this script, make sure you have the following dependencies installed:
+# `curl`, `jq`, `ttf-font-awesome`.
+#
+# NOTE: `inbox-reddit` can also be used outside of polybar by replacing the
+# following line: echo -e "%{F#cb4b16} %{F-}$unread"
+# by: echo " $unread"
+#
+# WARNING: in order to easily share this script without security risk, it is
+# recommended to place the JSON part containing the sensitive information in a
+# separate GPG file.
 
-url="your url here"
+# Replace this path by yours.
+reddit_gpg="/home/$USER/Sync/shared/.reddit-json.gpg"
```

Comment:
Can you explain what the content of your `.reddit-json.gpg` is?

Reply:
It is best to avoid making the JSON link clear because it contains sensitive data, so it is better to use a GPG file that contains this link in the form:
`<reddit_ID>&user=<reddit_username>`.

This allows you to share your script with GitHub in complete peace of mind.